build_tools/bazel_to_cmake/bazel_to_cmake_converter.py

```diff
@@ -226,10 +226,123 @@ def exports_files(self, *args, **kwargs):
   def td_library(self, *args, **kwargs):
     pass
 
-  # Technically we could do something with a CMake equivalent but we have no use
-  # case.
-  def py_binary(self, *args, **kwargs):
-    pass
```

Comment:
This would be one of my only reservations for accepting a change like this into IREE - if we don't use a feature directly in the repo then it could break, or we might feel pressure to accept more patches that don't directly affect the project. Forking or creating a separate shared repo (we could even aim for https://github.com/google/bazel-to-cmake , since that isn't actively maintained) would be more sustainable.

I'd also want to add some documentation like what's at https://github.com/google/bazel-to-cmake#why-convert-bazel-to-cmake if this grows into something more projects use.